### PR TITLE
Cascade Layers: New task

### PR DESCRIPTION
**Pull request by estelle**
Add a new task to the cascade and inheritance building block.

### CODE REVIEW

File: learn/tasks/cascade/marking.md [hunk 1]
```diff
@@ -8,6 +10,18 @@ One possible solution is as follows:
 }
 ```
 
-There are two things the student needs to do in this task. First, you need to write a selector for the a element which is more specific than the selector used to turn the background blue. I have achieved this by using the id selector which has very high specificity.
+There are two things the student needs to do in this task. First, write a selector for the `a` element which is more specific than the selector used to turn the background blue. In this solution, this is achieved by using the `id` selector which has very high specificity.
```
Comment (estelle) [inline learn/tasks/cascade/marking.md:13]:
```suggestion
There are two things the student needs to do in this task. First, write a selector for the `a` element which is more specific than the selector used to turn the background powderblue. In this solution, this is achieved by using the `id` selector which has very high specificity.  
```

File: learn/tasks/cascade/marking.md [hunk 2]
```diff
@@ -8,6 +10,18 @@ One possible solution is as follows:
...
+
+```
+ @layer yellow, green, purple;
+```
+
+There is one thing the student needs to do in this task: change the order of precedence so the declaration for the desired color is in the last declared layer, which is what his solution shows.
 
-Then they need to remember that there are special keyword values for all properties. In this case I am using inherit to set the background back to be the same as its parent element.
\ No newline at end of file
+The student needs to remember that unlayered normal styles take precedence over layered normal styles. But, if all styles are within layers, as in the case of this task, styles in later declared layers take precedence over styles declared in earlier layers. Moving the purple layer to end means it has precedence over the green and yellow layers.
```
Comment (estelle) [inline learn/tasks/cascade/marking.md:27]:
```suggestion
The student needs to remember that unlayered normal styles take precedence over layered normal styles. But, if all styles are within layers as in the case of this task, styles in later declared layers take precedence over styles declared in earlier layers. Moving the purple layer to the end means it has precedence over the green and yellow layers.
```

File: learn/tasks/cascade/marking.md [hunk 3]
```diff
@@ -8,6 +10,18 @@ One possible solution is as follows:
...
+
+```
+ @layer yellow, green, purple;
+```
+
+There is one thing the student needs to do in this task: change the order of precedence so the declaration for the desired color is in the last declared layer, which is what his solution shows.
 
-Then they need to remember that there are special keyword values for all properties. In this case I am using inherit to set the background back to be the same as its parent element.
\ No newline at end of file
+The student needs to remember that unlayered normal styles take precedence over layered normal styles. But, if all styles are within layers as in the case of this task, styles in later declared layers take precedence over styles declared in earlier layers. Moving the purple layer to the end means it has precedence over the green and yellow layers.
```
Comment (sideshowbarker) [inline learn/tasks/cascade/marking.md:27]:
```suggestion
The student needs to remember that unlayered normal styles take precedence over layered normal styles. But, if all styles are within layers — as in the case of this task — styles in later declared layers take precedence over styles declared in earlier layers. Moving the purple layer to the end means it has precedence over the green and yellow layers.
```

File: learn/tasks/cascade/marking.md [hunk 4]
```diff
@@ -8,6 +10,18 @@ One possible solution is as follows:
 }
 ```
 
-There are two things the student needs to do in this task. First, you need to write a selector for the a element which is more specific than the selector used to turn the background blue. I have achieved this by using the id selector which has very high specificity.
+There are two things the student needs to do in this task. First, write a selector for the `a` element which is more specific than the selector used to turn the background powderblue. In this solution, this is achieved by using the `id` selector which has very high specificity.
```
Comment (sideshowbarker) [inline learn/tasks/cascade/marking.md:13]:
```suggestion
There are two things the student needs to do in this task. First, write a selector for the `a` element which is more specific than the selector used to turn the background powderblue. In this solution, this is achieved by using the `id` selector, which has very high specificity.  
```

**Review by sideshowbarker — APPROVED** [review]
Looks great to me, with minor copy edits

**Comment by teoli2003** [comment]
As these are new tasks, I think we should run Prettier on each of these files.

What do you think, @estelle?

**Comment by estelle** [comment]
The file that calls it has already been merged so I would like to get this merged as soon as possible. So feel free to run prettier, and then please feel free to commit any changes directly. Thank you.

**Comment by teoli2003** [comment]
Done.

**Pull request merged**